backend/schemas/source/base/ComplianceRequirementTypes.json

```diff
@@ -0,0 +1,8195 @@
+{

```

Comment:
**question**: Is this too limiting because the multiple letters have to be in alphabetical order? If I were an auditor, I don't know if I would think to put them in this order and if I didn't, would the error message be clear enough to tell me that I should do that?

Reply:
In previous workbooks from Census, this is how requirement types were input. No changes are being introduced here, except for the fact that the cells were not formatted as a list as they were in Census workbooks. So this ticket was just to rectify this by converting the cells back into a list format, just as they were before. Also, I have included a sheet in the workbook that contains the entire list, allowing users to refer to it if necessary.

Reply:
About the error, I am tagging @jadudm. As of now we have a generic error message for any invalid list input that reads `Not in the lookup list`. Should we make an exception here and use the message previously used by Census `Please enter a valid Type of Compliance Requirement. (One or more of ABCEFGHIJLMNP, in alphabetical order).` ?

Reply:
Oh, yes, by all means... error message improvements are a good idea. Ultimately, the `link` field in the error will let us link back to `fac.gov`, where we can write a more thorough story about what the error is about. But, in the meantime, feel free to update that error message so it tells the user exactly what is going on.

Reply:
Done!, I used the same message from Census workbook.